gpii/node_modules/xrandr/nodexrandr/nodexrandr.cc

```diff
@@ -39,6 +39,13 @@
 #endif
 
 using namespace v8;
+using v8::FunctionTemplate;
+using v8::Local;
+using v8::Object;
+using Nan::GetFunction;
+using Nan::New;
+using Nan::Set;
+
 using namespace std;
```

Comment:
See http://stackoverflow.com/questions/1452721/why-is-using-namespace-std-in-c-considered-bad-practice - I think we should axe all "using"s